"Unsupported class file major version 60" Error when running mutation test

```
tehtea@somePC spring-data-elasticsearch % ./mvnw org.pitest:pitest-maven:mutationCoverage                                         
[INFO] Scanning for projects...
[INFO] 
[INFO] ---------< org.springframework.data:spring-data-elasticsearch >---------
[INFO] Building Spring Data Elasticsearch 5.0.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- pitest-maven:1.5.2:mutationCoverage (default-cli) @ spring-data-elasticsearch ---
[WARNING] The artifact xml-apis:xml-apis:jar:2.0.2 has been relocated to xml-apis:xml-apis:jar:1.0.b2
[INFO] Found plugin : Default csv report plugin
[INFO] Found plugin : Default xml report plugin
[INFO] Found plugin : Default html report plugin
[INFO] Found plugin : Static initializer code detector plugin
[INFO] Found plugin : Static initializer filter plugin
[INFO] Found plugin : Excluded annotations plugin
[INFO] Found plugin : Try with resources filter
[INFO] Found plugin : Inlined finally block filter plugin
[INFO] Found plugin : Implicit null check filter
[INFO] Found plugin : Method reference null check filter
[INFO] Found plugin : For each loop filter
[INFO] Found plugin : Enum constructor filter
[INFO] Found plugin : Logging calls filter
[INFO] Found plugin : Infinite for loop filter
[INFO] Found plugin : Long running iterator loop filter
[INFO] Found plugin : For loop counter filter
[INFO] Found plugin : Kotlin junk mutations filter
[INFO] Found plugin : Max mutations per class limit
[INFO] Found plugin : Equals shortcut equivalent mutant filter
[INFO] Found plugin : Trivial return vals equivalence filter
[INFO] Found plugin : Mutant export plugin
[INFO] Found shared classpath plugin : Default mutation engine
[INFO] Found shared classpath plugin : JUnit 5 test framework support
[INFO] Found shared classpath plugin : JUnit plugin
[INFO] Found shared classpath plugin : TestNG plugin
[INFO] Adding org.pitest:pitest-junit5-plugin to SUT classpath
[INFO] Adding org.pitest:pitest to SUT classpath
[INFO] Mutating from /Users/tehtea/spring-data-elasticsearch/target/classes
[INFO] Defaulting target tests to match packages in test build directory
18:54:54 PIT >> INFO : Verbose logging is disabled. If you encounter a problem, please enable it before reporting an issue.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.997 s
[INFO] Finished at: 2022-03-26T18:54:54+08:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.pitest:pitest-maven:1.5.2:mutationCoverage (default-cli) on project spring-data-elasticsearch: Execution default-cli of goal org.pitest:pitest-maven:1.5.2:mutationCoverage failed: Unsupported class file major version 60 -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
```

--- a/pom.xml
+++ b/pom.xml
@@ -435,7 +435,7 @@
 			<plugin>
 				<groupId>org.pitest</groupId>
 				<artifactId>pitest-maven</artifactId>
-				<version>1.5.2</version>
+				<version>1.7.5</version>
 				<dependencies>
 					<dependency>
 						<groupId>org.pitest</groupId>
@@ -444,6 +444,9 @@
 					</dependency>
 				</dependencies>
 				<configuration>
+					<jvmArgs>
+						<jvmArg>-XX:+AllowRedefinitionToAddDeleteMethods</jvmArg>
+					</jvmArgs>
 					<excludedGroups>integration-test</excludedGroups>
 					<targetClasses>
 						<param>org.springframework.data.elasticsearch.core.geo.*</param>
@@ -460,7 +463,7 @@
 				<groupId>org.asciidoctor</groupId>
 				<artifactId>asciidoctor-maven-plugin</artifactId>
 			</plugin>
-		</plugins>
+        </plugins>
 	</build>
 
 	<profiles>

